src/reader/mod.rs

```diff
@@ -211,7 +211,7 @@ macro_rules! read_event_impl {
     ) => {{
         let event = loop {
             break match $self.state.state {
-                ParseState::Init => { // Go to OpenedTag state
```

Comment:
The commit message says "renamed for brevity".  I'm not sure that brevity is the correct word since these names are longer than the original ones.

I'm fine with the new names, though.

Reply:
Ah, yes, I confused it with "clarity"

Reply:
I fixed commit text in force push